lib/analyze_config_file_diffs_task.rb

```diff
@@ -71,6 +73,17 @@ def analyze(description)
     end
   end
 
+  def check_for_zypper_version
+    supported_zypper_version = 9.17
+    cheetah = Cheetah.run("zypper", "--version", stdout: :capture)
+    current_zypper_version = /[1-9]\.([1-9]\.[1-9].*)/.match(cheetah)
+    if LocalSystem.os["name"] == "openSUSE 13.1 (Bottle)" &&
+      supported_zypper_version > current_zypper_version[1].to_f
```

Comment:
Use 4 (not 2) spaces for indenting a condition in an `if` statement spanning multiple lines.